src/db/tables/featureFlags.ts

```diff
@@ -42,8 +42,8 @@ async function addFeatureFlag(flag: FeatureFlag) {
     is_enabled: flag.isEnabled,
     description: flag.description,
     dependencies: flag.dependencies,
-    allow_list: flag.allowList,
-    wait_list: flag.waitList,
+    allow_list: flag.allowList?.map((e) => e.trim()).filter((e) => !!e),
+    wait_list: flag.waitList?.map((e) => e.trim()).filter((e) => !!e),
```

Comment:
Q: is there a way/need to merge `.map().filter()` to reduce looping over arrays? Or is the data set so small we don't really need to worry?

But wasn't sure if something like this would work:
```js
flag.allowList?.filter((e) => !!e.trim()),
```

or a slightly uglier [and untested] `.reduce()`:
```js
flag.allowList.reduce((acc, e) => {
  e = e.trim();
  return !!e ? [...acc, e] : acc;
}, []);

Reply:
Thanks @pdehaan,  great suggestion

Reply:
actually, sorry, the filter one won't work because we need the trim() to apply to the emails

Reply:
The reduce function actually returns a string, we need string[]

Reply:
K I've modified the reduce, I think that's the right approach